reference/core/ingress-controller.md

```diff
@@ -107,7 +107,7 @@ For example, this `Ingress` resource
 
 ```yaml
 ---
-apiVersion: extensions/v1beta1
+apiVersion: extensions/v1
```

Comment:
This sample is out of date with the `v1` API.
@alexgervais to provide a good and valid example.